docs/_docs/live-update/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Live Update'
+layout: page
+permalink: docs/live-update
+nav_order: 3
+has_children: true
+has_toc: false
+---
+
+{% include toc.md %}
+
+# Live Update
+
+ The Live Update feature automatically downloads and displays Figma designs as
+they're updated in Figma. This enables designers, engineers, and product
+managers to quickly refine designs and try out new ideas in an emulator, on a
+test bench, or in a car.
+
+## Automatic Figma design syncing {#AutomaticSyncing}
+
+First, you must grant Live Update feature access to your account on Figma.com
+using a personal access token. When access is granted, Live Update automatically
+downloads the latest updates to your Figma designs, allowing the DesignCompose
+library to render them on screen.
+
+Learn how to set it up on [Setting Up Figma Authentication][1].
```

Comment:
Does this link to the "Live Update Setup" on the left menu? I might name it the same if it does.

Reply:
Done